sentences/ur/_common.yaml

```diff
@@ -1,12 +1,107 @@
 language: ur
 responses:
   errors:
-    no_intent: TODO Sorry, I couldn't understand that
-    no_area: TODO No area named {{ area }}
-    no_domain: TODO {{ area }} does not contain a {{ domain }}
-    no_device_class: TODO {{ area }} does not contain a {{ device_class }}
-    no_entity: TODO No device or entity named {{ entity }}
-    handle_error: TODO An unexpected error occurred while handling the intent
-lists: {}
-expansion_rules: {}
-skip_words: []
+    no_intent: "مجھے سمجھ میں نہیں آیا"
+    no_area: "کسی جگہ کا نام  {{ area }} نہیں ھے"
+    no_domain: "{{ area }} میں {{ domain }} نہیں ہے"
+    no_device_class: "{{ area }} میں {{ device_class }} نہیں ہے"
+    no_entity: "کوئی ڈوائس کا نام {{ entity }} نہیں ھے "
+    handle_error: "آپ کے جواب کو سنبھالتے وقت کچھ عجیب مسئلہ آگیا"
+lists:
+  color:
+    values:
+      # white
+      - in: "سفید"
+        out: "white"
+      - in: "white"
+        out: "white"
+      # black
+      - in: "سیاہ"
+        out: "black"
+      - in: "کالا"
+        out: "black"
+      - in: "black"
+        out: "black"
+      # red
+      - in: "سرخ"
+        out: "red"
+      - in: "لال"
+        out: "red"
+      - in: "red"
+        out: "red"
+      # orange
+      - in: "اورنج"
+        out: "orange"
+      - in: "نارنجی"
+        out: "orange"
+      - in: "orange"
+        out: "orange"
+      # yellow
+      - in: "پیلا"
+        out: "yellow"
+      - in: "yellow"
+        out: "yellow"
+      # green
+      - in: "سبز"
+        out: "green"
+      - in: "ہرا"
+        out: "green"
+      - in: "green"
+        out: "green"
+      # blue
+      - in: "نیلا"
+        out: "blue"
+      - in: "blue"
+        out: "blue"
+      # purple
+      - in: "جامنی"
+        out: "purple"
+      - in: "purple"
+        out: "purple"
+      # pink
+      - in: "پنک"
+        out: "pink"
+      - in: "گلابی"
+        out: "pink"
+      - in: "pink"
+        out: "pink"
+      # brown
+      - in: "براؤن"
+        out: "brown"
+      - in: "brown"
+        out: "brown"
+      # sky blue
+      - in: "آسمانی"
+        out: "sky blue"
+      - in: "sky blue"
+        out: "sky blue"
+
+  brightness:
+    range:
+      type: "percentage"
+      from: 0
+      to: 100
+  temperature:
+    range:
+      type: "temperature"
+      from: 0
+      to: 100
+  temperature_unit:
+    values:
+      - "celsius"
+      - in: "c"
+        out: "celsius"
+
+expansion_rules:
+  name: "{name} [کو] [کے] [کا]"
+  area: "{area} [کا] [کے] [کو] [میں]"
+  temp: "(temperature | درجہ حرارت | ٹیمپریچر)"
+  temperature: "{temperature} [ڈگری] [degree] [درجہ حرارت] [{temperature_unit}]"
+  brightness: "{brightness:brightness_pct} [percent | پرسنٹ]"
+  set: "[میں] [پر] (کر دیں | سیٹ کردو | کردو | رکھلو | دالدو | دالو | سیٹ کرو | کرو | کر)"
+  all: "(تمام | سب | ساری | سارے)"
```

Comment:
Expansion rules that do not wrap a slot should be written in the native language.

Reply:
I am a bit confused about what you mean by "wrap a slot". Do you want me to write set in urdu?

Reply:
BTW I would like to be a language leader as well